.github/instructions/php-block.instructions.md

```
@@ -1,54 +1,70 @@
+-   instructions
+    categories:
+-   documentation
+-   instructions
+-   guides
+    applyTo: "\*_/_.php"
+
+---
+
+# PHP Block & Theme Setup Instructions
+
+-   Follow WordPress coding standards: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)